### ISSUE TITLE
False-Positive prop-types Error?

### ISSUE THREAD

**Issue by epmatsw** [yannickcr/eslint-plugin-react#255]
The following code:
```
      Copyable.prototype.render = function() {
        return React.createElement(ReactZeroClipboard, {
          "text": this.props.textToCopy
        });
      };
```
gives the following warning:
```
error  'props' is missing in props validation        react/prop-types
```

My propTypes are set up correctly (as far as I know), and even if they weren't, it's weird that it complains about `props`:
```
Copyable.propTypes = {
        buttonText: React.PropTypes.string.isRequired,
        textToCopy: React.PropTypes.string.isRequired,
        copyClassName: React.PropTypes.string,
        htmlToCopy: React.PropTypes.string
      };
```
